hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/auth/CredentialProviderListFactory.java

```diff
@@ -233,6 +236,11 @@ public static AWSCredentialProviderList buildAWSProviderList(
             key, className, mapped);
         className = mapped;
       }
+      if (awsCredsMappedClasses != null && awsCredsMappedClasses.containsKey(className)) {
```

Comment:
make an `else` unless we really want to support remapping of the standard map to different values. Which I suppose we might...